package com.rashmitech.exceptiondemo;

public class Arrayoutofboundexception {

	public static void main(String[] args) {
		int[] arr=new int[4];
		arr[0]=11;
		arr[1]=12;
		arr[2]=13;
		arr[3]=14;
		
		System.out.println(arr[4]);


	}

}
//Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 4
	at com.rashmitech.exceptiondemo.Arrayoutofboundexception.main(Arrayoutofboundexception.java:12)
